app/main.py

```diff
@@ -1,6 +1,22 @@
+import requests
+import os
+
+
+URL = "https://api.weatherapi.com/v1/current.json"
+FILTERING = "Paris"
```

Comment:
it is just a `CITY` or `LOCATION` isn't it?:)

Reply:
Well, it`s according to best practice from the checklist, so ... 
![image](https://user-images.githubusercontent.com/123254785/232592528-02ef9e70-17b1-48ac-b276-a020b3417fd2.png)

Reply:
I guess we need to update checklist:)

Reply:
Actually I disagree, I mean when you filter queryset by exact parameter it's better to see `filter`, not `city` or `location`. Sure, best would be something like filter_by_city, but it's pretty redundant since we apply only 1 filter)